src/Clothingabstactclass.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Clothing abstract class.
+ */
+public abstract class Clothingabstactclass {
+
+    /**
+     * Would represent the brand of the clothing.
+     */
+    private String brand;
+    /**
+     * Would represent the size of the clothing.
+     */
+    private String size;
+    /**
+     * Would represent the color of the clothing.
+     */
+    private String color;
+
+    /**
+     * Would represent the material of the clothing.
+     */
+    private String material;
+
+    /**
+     * Clothing component with brand size and color.
+     *
+     * @param brand
+     *            the brand of the clothing
+     * @param size
+     *            size of the clothing item
+     * @param color
+     *            color of the clothing item
+     * @param material
+     *            material of the clothing item
+     */
+    public Clothingabstactclass(String brand, String size, String color,
```

Comment:
Same goes for constructors. The methods will only have access to kernel methods.